docs/testplan/MACsec-neighbor-reboot.md

```diff
@@ -0,0 +1,54 @@
+- [Overview](#overview)
+    - [Scope](#scope)
+    - [Testbed](#testbed)
+- [Setup configuration](#setup-configuration)
+- [Test cases](#test-cases)
+
+# Test name
+
+Macsec Neighbor Reboot
+
+## Overview
+
+The goal of this test is to verify that the macsec neighbor relationship is established after the neighbor is rebooted.
+
+### Scope
+
+The test is targeting a running SONIC system with fully functioning configuration. The purpose of the test is to verify
+the behavior of a link when a neighbor reboots while Macsec is Active and ensure the link returns to functioning state.
```

Comment:
This is better. Yet, I don't think you are verifying the behavior of a "link" in this test case. What you do is to verify the behavior of a pair of MACsec peers when the neighbor reboots.

Reply:
I updated this section.